app/services/csv_exporter.rb

```diff
@@ -1,7 +1,8 @@
 require 'csv'
 
 module CsvExporter
-  def self.export(resources, columns, header = default_header(columns))
+  def self.export(resources, columns, header = nil)
+    header ||= default_header(columns)
```

Comment:
can't use default value here because the csv responder may pass a nil which will cause the default value to be ignored.